### PR TITLE
{Core} refine local context message

### LINKED ISSUE #14015: [local context] refine error message
> ### `az feedback` auto-generates most of the information requested below, as of CLI version 2.0.62
Refine messages of local context when saving per PM feedback, below is an example:
"Your preference of: –name AzureFunctionsQuickstart-rg and –location westeurope are now saved to local context. To learn more, type in az local-context –help "

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Environment summary**

**Additional context**

### PR DESCRIPTION
**Pull request by arrownj**
**Description**  
This PR is to fix #14015 .

PM want to refine some of local context message. 

**Testing Guide**  

`az local-context on`

> Command group 'local-context' is experimental and not covered by customer support. Please use with discretion.
Local context is turned off, you can run `az local-context on` to turn it on.

`az group create -g xxj -l westus2`
> Local context is turned on. Its information is saved in working directory D:\code\cli\azure-cli. You can run `az local-context off` to turn it off.
Your preference of: --name: xxj, --location: westus2 are now saved to local context. To learn more, type in az local-context --help

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
local context

File: src/azure-cli-core/azure/cli/core/__init__.py [hunk 1]
```diff
@@ -147,7 +147,8 @@ def save_local_context(self, parsed_args, argument_definitions, specified_argume
             args_str = []
             for name, value in local_context_args:
                 args_str.append('{}: {}'.format(name, value))
-            logger.warning('Command argument values saved to local context: %s', ', '.join(args_str))
+            logger.warning('Your preference of: %s now saved to local context. To learn more, type in az '
+                           'local-context --help', ', '.join(args_str) + ' is' if len(args_str) == 1 else ' are')
```
Comment (jiasli) [inline src/azure-cli-core/azure/cli/core/__init__.py:151]: The colon seems weird to me. How about

```suggestion
            logger.warning('Your preference of '%s' are now saved to local context. To learn more, type in az '
                           'local-context --help', ', '.join(args_str) + ' is' if len(args_str) == 1 else ' are')
```

Comment (jiasli) [reply src/azure-cli-core/azure/cli/core/__init__.py:151]: We usually use backticks (`` ` ``) to denote commands. See https://github.com/Azure/azure-cli/pull/13060.

**Comment by yonzhan** [comment]
Core

**Pull request merged**